src/App.tsx

```diff
@@ -5,16 +5,29 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (movie: Movie): void => {
+    const isInList = movies.some(item => item.imdbId === movie.imdbId);
```

Comment:
It's better not to use 'isInList' as a variable name. The name should reflect its purpose clearly. You can use something like 'isMovieInList' which provides more context.